turbo.json

```diff
@@ -0,0 +1,22 @@
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
```

Comment:
Can it depend on itself?

Reply:
The `^` means its dependencies' `build` needs to be completed first
https://turborepo.org/docs/reference/configuration#dependson